src/store/actions.js

```diff
@@ -197,6 +202,17 @@ export default {
 			})
 		)
 	},
+	changeFolderSubscription({commit}, {accountId, folderId, subscribed}) {
+		logger.debug('Toggle subscription for account: ' + accountId + ' folder ' + folderId + ' to ' + subscribed)
+		return changeFolderSubscription(accountId, folderId, subscribed).then((res) => {
+			var isSubscribed = res.subscribed
```

Comment:
you can use `const`. But actually it would be also easier if you use `isSubscribed: res.subscribed` below

Reply:
Great, did not know it works this way.